tree/dataframe/inc/ROOT/RDFInterface.hxx

```diff
@@ -1654,7 +1648,7 @@ private:
       using RCustomColumnBasePtrMap_t = std::map<std::string, std::shared_ptr<RCustomColumnBase>>;
 
       auto loopManager = GetLoopManager();
-      RDFInternal::CheckCustomColumn(name, loopManager->GetTree(), *(fCustomColumns.fCustomColumnsNames),
+      fCustomColumns.CheckCustomColumn(name, loopManager->GetTree(),
                                      fDataSource ? fDataSource->GetColumnNames() : ColumnNames_t{});
```

Comment:
@imaxoi i'm not sure about this...it shouldn't be the job of `fCustomColumns` to check whether `name` is present in the TTree or the datasource, it doesn't make much sense..?

Reply:
Moved back in a0deeb35e81cb57591363dc3fd8fbe76264611c2